Content.Client/Players/PlayTimeTracking/JobRequirementsManager.cs

```diff
@@ -56,11 +57,9 @@ private void RxRoleBans(MsgRoleBans message)
     {
         _sawmill.Debug($"Received roleban info containing {message.Bans.Count} entries.");
 
-        if (_roleBans.Equals(message.Bans))
-            return;
-
```

Comment:
Any reason this was removed? If it's just sending the same data nothing should be updating.

Reply:
Mistake I'm pretty sure, added back.